diff --git a/src/Plugin/Service/Type/Analyse/Drupal/ProjectRelease.php b/src/Plugin/Service/Type/Analyse/Drupal/ProjectRelease.php
index 899eb9b..4f85f5d 100644
--- a/src/Plugin/Service/Type/Analyse/Drupal/ProjectRelease.php
+++ b/src/Plugin/Service/Type/Analyse/Drupal/ProjectRelease.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace Drupal\update;
+namespace App\Plugin\Service\Type\Analyse\Drupal;
 
 use Symfony\Component\Validator\Constraints\Choice;
 use Symfony\Component\Validator\Constraints\Collection;
@@ -107,7 +107,7 @@ final class ProjectRelease {
    * @param array $release_data
    *   The project release data as returned by update_get_available().
    *
-   * @return \Drupal\update\ProjectRelease
+   * @return ProjectRelease
    *   The ProjectRelease instance.
    *
    * @throws \UnexpectedValueException
